### DISCUSSION TITLE
Doesn`t show clock

### DISCUSSION THREAD

**Discussion by rreessttart** [valinet/ExplorerPatcher#2521] [Q&A] [answered]
I downloaded the application, everything was shown to me, but at one point I couldn’t find how to change the style of the main menu in the Windows section so that I click on Windows and Windows 10 opens there, I decided to reset the settings, after which my clock does not show
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/72eda919-b555-4c15-817e-25dff497aa2e)

**Comment by rreessttart** [comment]
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/c1e38b24-f8fc-46bc-85cc-7bc5ec5690b9)
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/af4f6bf2-fdd0-4e03-8156-c58f1996070a)
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/a44afe14-5173-4253-96bb-b660decefe61)
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/4052bf5b-b705-4877-b8d5-eafbe4434902)
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/57f055ef-60c1-4274-bc14-d7d80f9e55df)
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/b0ad4c49-74a2-43d3-8299-bac15bc595fd)
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/147c157d-f1c1-4547-a668-5d2a69918693)
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/be8b0f8c-4f89-426c-ae92-866ceab4c4ab)
![image](https://github.com/valinet/ExplorerPatcher/assets/152426652/9a4951ad-e79e-47fb-80da-cf2406e634c1)


There my setting, maybe i don`t turn something

**Comment by Willie982** [comment, marked as answer]
![EP1](https://github.com/valinet/ExplorerPatcher/assets/151062840/cf54e398-2159-4c1d-b8b8-51b0dd81485a)

**Comment by pyrates999** [comment]
not a bug with EP: https://superuser.com/questions/1417624/how-can-i-hide-the-clock-from-windows-10-taskbar

**Comment by highlanderc** [comment]
I'm having the same issue... clock and system tray are gone.

**Reply by pyrates999** [reply]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.

Please include any other software you have installed that modifies the UI of windows.

**Reply by Amrsatrio** [reply]
@pyrates999 Remember that MS nuked the system tray in 25921? So that build and the builds after that don't have the system tray. I know that the system tray reimplementation is almost done, but it is not fully polished yet.

Now that a lot of people are asking about this this is where I'm becoming very tired at it, and I think you should ignore them too, for your own good.
